src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/NotifyService.kt

```diff
@@ -313,6 +314,34 @@ class NotifyService(
     }
   }
 
+  internal fun sendInitialComAllocationEmail(
+    emailAddress: String?,
+    comName: String,
+    offenderName: String,
+    crn: String?,
+    licenceId: String,
+  ) {
+    if (emailAddress.isNullOrBlank()) {
+      log.error("Notification failed (sendInitialComAllocationEmail) - Email address not present for licence Id $licenceId")
+      return
+    }
+    if (crn.isNullOrBlank()) {
```

Comment:
suggestion: it's not possible to create a licence without a CRN so we don't need to explicitly handle it